forest/cli/src/cli/wallet_cmd.rs

```diff
@@ -241,3 +278,191 @@ impl WalletCommands {
         }
     }
 }
+
+fn format_balance_string(
```

Comment:
Tests are great but this function still needs some documentation.

Reply:
Added the docs

Reply:
I meant documentation at the top of the function.

Reply:
Added it too